### DISCUSSION TITLE
Quota and limits

### DISCUSSION THREAD

**Discussion by mr-falken** [HewlettPackard/squest#429] [Ideas]
Hi,
about the topic, while it's possibile to handle resource creation approve/denial process using hooks and api, I think it might be usefull to have requests blocked for some specific service when a specific quota is reached.

Associating one or more service to one or more resource type (i.e. vCPU) would allow squest to disable that kind of service is the user is over quota for that specific resource.

**Comment by Sispheor** [comment]
Hi, Thanks for proposing the idea.
It would be technically possible. But we need to work on the RBAC before.
I propose to keep this one here and discuss about it when everything is ready .

**Comment by Sispheor** [comment]
Closing this as the quota is going to be refactored with V2
